CreateDB

```diff
@@ -13,32 +17,33 @@ CREATE TABLE Chat_Group (
 
 CREATE TABLE Chat_Members (
     cid INTEGER REFERENCES Chat_Group(cid),
-    uid INTEGER REFERENCES Users(uid)
+    uid INTEGER REFERENCES Users(uid),
+    joined_on TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP
 );
 
-CREATE TABLE Parcels (
-    pid serial PRIMARY KEY,
+CREATE TABLE Messages (
+    mid serial PRIMARY KEY,
     cid INTEGER REFERENCES Chat_Group(cid),
     uid INTEGER REFERENCES Users(uid),
     created_on TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP
 );
 
 CREATE TABLE Photos (
```

Comment:
How about calling this table Multimedia in case we are able to add videos for bonus points.